Mage.Sets/src/mage/cards/t/TamiyoTheMoonSage.java

```diff
@@ -62,7 +63,8 @@ public TamiyoTheMoonSage copy() {
     }
 }
 
-class TappedCreaturesControlledByTargetCount implements DynamicValue {
+enum TappedCreaturesControlledByTargetCount implements DynamicValue {
```

Comment:
So this was transformed as an enum there, I can remove it and group it with the other class => enum if you really want to.